sdk/identity/identity-cache-persistence/.gitignore

```diff
@@ -1 +1 @@
-!test/assets/cert.pem
+!test/assets/fake-cert.pem
```

Comment:
Why do we need this? Is this file often written to that we want to not check it in?

Reply:
I wonder if this can be removed. Let me ask @willmtemple

Reply:
I'd suggest we remove the entire file. We try to avoid scoped gitignore files in most cases anyway.

Reply:
The issue here is that I don’t think this specific file exists at all 🤔 I don’t think tests are using it. I could be missing something.

Reply:
Will says I can delete this, so I will!